extensions/chrome/manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Nextcloud Screensharing",
+  "description": "Screensharing utility for the Nextcloud video calls app.",
+  "version": "0.0.1",
+  "manifest_version": 2,
+  "minimum_chrome_version": "34",
+  "icons": {
+  },
+  "permissions": [
+    "desktopCapture"
+  ],
+  "background": {
+    "scripts": ["background.js"]
+  },
+  "content_scripts": [ {
+    "js": [ "content.js" ],
+    "matches": [ "https://replace-with-your-hostname/*" ]
```

Comment:
This one actually would allow wildcard hostnames, it's the `matches` below that doesn't allow it.
I'll look if there is another way to communicate between website and the extension, or if we have to deal with the "one extension per hostname".